common/beerocks/bwl/dwpal/mon_wlan_hal_dwpal.cpp

```diff
@@ -1080,26 +1080,29 @@ bool mon_wlan_hal_dwpal::process_dwpal_nl_event(struct nl_msg *msg)
     case Event::Channel_Scan_Dump_Result: {
```

Comment:
Commit message is too long and not clear.
Suggestion:
>When a triggered scan completes, an empty 'Channel_Scan_Dump_Result' event
sent to signal the scan is completed and that results are ready.
At this point, if dump-results is requested, it will result with a
series of 'Channel_Scan_Dump_Result' events with the results.
Fix current logic by treating the first 'Channel_Scan_Dump_Result' as results-ready
and trigger the dump-result sequence.